File: experiments/016-LanguageComparisonRepeated/cells/java-gradle_1-Empty_free/2019-2c-parcial2-masked/20260917-061856-52134/output/project/src/main/java/surveycrawler/Boulder.java
package surveycrawler;

public class Boulder extends Ground {

    // moving

    @Override
    public void moveCrawlerTowards(SurveyCrawler aCrawler, Point aDirection) {
        aCrawler.signalBoulderFound();
    }

    // facing

    @Override
    public void turnCrawler(SurveyCrawler aCrawler, Runnable aTurn) {
        aCrawler.signalBoulderFound();
    }
}

File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2019-2c-parcial2-masked/20260921-202806-59410/output/project/src/surveycrawler/boulder.py
from surveycrawler.ground import Ground


class Boulder(Ground):

    # moving

    def move_crawler_towards(self, a_crawler, a_direction):
        a_crawler.signal_boulder_found()

    # facing

    def turn_crawler(self, a_crawler, a_turn):
        a_crawler.signal_boulder_found()
